plugins/rapid7_insightidr/help.md

```diff
@@ -40,6 +47,411 @@ Example input:
+
+Example output:
+
+```
+{
+  "investigation": {
+    "assignee": {
+      "email": "user@example.com",
+      "name": "Ellen Example"
+    },
+    "created_time": "2018-06-06T16:56:42Z",
+    "disposition": "BENIGN",
+    "first_alert_time": "2018-06-06T16:56:42Z",
+    "last_accessed": "2018-06-06T16:56:42Z",
+    "latest_alert_time": "2018-06-06T16:56:42Z",
+    "organization_id": "174e4f99-2ac7-4481-9301-4d24c34baf06",
+    "priority": "CRITICAL",
+    "rrn": "rrn:example",
+    "source": "ALERT",
+    "status": "OPEN",
+    "title": "Example Title"
+  }
+}
+```
+
+#### Update Investigation
+
+This action allows to update existing investigation.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|disposition|string|None|False|Investigation's disposition|['', 'BENIGN', 'MALICIOUS', 'NOT_APPLICABLE']|BENIGN|
+|email|string|None|False|A user's email address for investigation to be assigned|None|user@example.com|
+|id|string|None|True|The identifier of Investigation to be update (ID or RRN)|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+|priority|string|None|False|Investigation's priority|['', 'UNSPECIFIED', 'LOW', 'MEDIUM', 'HIGH', 'CRITICAL']|LOW|
+|status|string|None|False|Investigation's status|['', 'OPEN', 'INVESTIGATING', 'CLOSED']|OPEN|
+|title|string|None|False|Investigation's title|None|Example Title|
+
+Example input:
+
+```
+{
+  "disposition": "BENIGN",
+  "email": "user@example.com",
+  "id": "rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111",
+  "priority": "LOW",
+  "status": "OPEN",
+  "title": "Example Title"
+}
+```
+
+##### Output
+
+|Name|Type|Required|Description|
+|----|----|--------|-----------|
+|investigation|investigation|True|The body of the specified investigation|
+
+Example output:
+
+```
+{
+  "investigation": {
+    "assignee": {
+      "email": "user@example.com",
+      "name": "Ellen Example"
+    },
+    "created_time": "2018-06-06T16:56:42Z",
+    "disposition": "BENIGN",
+    "first_alert_time": "2018-06-06T16:56:42Z",
+    "last_accessed": "2018-06-06T16:56:42Z",
+    "latest_alert_time": "2018-06-06T16:56:42Z",
+    "organization_id": "174e4f99-2ac7-4481-9301-4d24c34baf06",
+    "priority": "CRITICAL",
+    "rrn": "rrn:example",
+    "source": "ALERT",
+    "status": "OPEN",
+    "title": "Example Title"
+  }
+}
+```
+
+#### Search Investigation
+
+This action allows to search for investigations that match the given criteria.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|end_time|date|None|False|The ending time when investigations were created|None|2020-09-06 12:07:55.136667|
+|index|integer|0|True|The optional zero-based index of the page to retrieve. Must be an integer greater than or equal to 0|None|1|
```

Comment:
Is this required? Description contradicts the Default.

Reply:
That field was required in the actions made before (same as `size` field) that update - so I left that same way as before for clarity 😃